nixos/modules/services/hardware/udev.nix

```diff
@@ -85,7 +85,7 @@ let
       for i in $import_progs $run_progs; do
         if [[ ! -x $i ]]; then
           echo "FAIL"
-          echo "$i is called in udev rules but not installed by udev"
+          echo "$i is called in udev rules but is not executable"
```

Comment:
I think it could be either so something like ```is called in udev rules but does not exist or is not executable```

Reply:
I'll leave "is not executable" first, as that's what is being checked here.